cmd/cagent/toast_fallback.go

```diff
@@ -0,0 +1,21 @@
+// +build !windows
+
+package main
+
+import (
+	"errors"
+
+	"github.com/cloudradar-monitoring/cagent"
+)
+
+func sendErrorNotification(_, _ string) error {
+	return errors.New("Implemented only or Windows")
```

Comment:
Can you make error string begin with lower-case

Reply:
this error can't be reached but fixed